src/main/java/com/ldtteam/blockui/util/texture/IsOurTexture.java

```diff
@@ -0,0 +1,11 @@
+package com.ldtteam.blockui.util.texture;
+
+import net.minecraft.client.renderer.texture.AbstractTexture;
+
+public final class IsOurTexture
+{
+    public static final boolean isOur(final AbstractTexture texture)
```

Comment:
Why does this matter? =D

Reply:
To avoid cyclic reload when u load sprite outofjar:
a) image asks sprite to ensure sprite is loaded -> sprite
b) blit asks outofjar to ensure outofjar is loaded -> outofjar
c) repeat a)